### ISSUE TITLE
GirsLite-1.0.3-nano-flasher.sh doesn't work on Macs

### ISSUE THREAD

**Issue by adrcunha** [bengtmartensson/AGirs#59]
Because the OS check is hardcoded incorrectly:

```
# Mac defaults
if [ Linux == Darwin ]; then
  AVRDUDE=/Applications/Arduino.app/Contents/Java/hardware/tools/avr/bin/avrdude
  AVRDUDE_CONF=/Applications/Arduino.app/Contents/Java/hardware/tools/avr/etc/avrdude.conf
fi
```

**Comment by adrcunha** [comment]
Root cause: `mkflasher.sh` is missing an escape here:

https://github.com/bengtmartensson/AGirs/blob/eee6a2119cda997ad8310412828682357effaa7f/tools/mkflasher.sh#L68

which in turns hardcodes the OS the script is running on into the generated flasher script.

**Comment by bengtmartensson** [comment]
Was fixed by PR #60. Thank you!
